cluster-bootstrap/src/test/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinatorSpec.scala

```diff
@@ -102,6 +102,59 @@ class BootstrapCoordinatorSpec extends WordSpec with Matchers with BeforeAndAfte
     }
   }
 
+  "BootstrapCoordinator target filtering" should {
+    "not filter when port-name is set" in {
```

Comment:
Perhaps we should add comments clarifying in which situations the different behaviors are expected? I guess:

```suggestion
    // For example when using DNS SRV-record-based discovery
    "not filter when port-name is set" in {
```